sources/packages/backend/workflow/test/test-utils/factories/assessment-consolidated-data.ts

```diff
@@ -97,5 +99,28 @@ function setDefaultAssessmentConsolidatedData(): AssessmentConsolidatedData {
     partner1StudentStudyWeeks: null,
     partner1TotalIncome: null,
     partner1CRAReportedIncome: null,
+    assessmentId: null,
+    applicationExceptionStatus: null,
   } as AssessmentConsolidatedData;
 }
+
+export function createFakeConsolidatedFulltimeData(
```

Comment:
There was a decision on a previous PR to have all methods commented on, even these types of methods.